### ISSUE TITLE
Doesn't work with Next.js (server side rendering)

### ISSUE THREAD

**Issue by mattjis** [GoncharukBro/react-smart-masonry#1]
```
Server Error
ReferenceError: window is not defined
```

[Possible solutions](https://medium.com/wesionary-team/window-object-in-nextjs-1e505349c6f5)

**Comment by mattjis** [comment]
I can get it to work by importing it like this:
```javascript
import dynamic from "next/dynamic"
const Masonry = dynamic(() => import("react-smart-masonry"), { ssr: false })
```

But would love for it to work natively with SSR!

**Comment by GoncharukBro** [comment]
Released patch (v2.0.2), update the package to the latest version.

**Comment by mattjis** [comment]
There is a warning left over. Not really a big issue though:

```
Warning: Prop 'style' did not match.
Server: "box-sizing:border-box;display:block;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;padding-top:80%" 
Client: "box-sizing:border-box;display:block;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;padding-top:121.29144851657941%"
```
